app/views/catalog/_universal_viewer_issue.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if resource.class.can_have_manifests? && resource.try(:member_ids).present? && !Wayfinder.for(resource).coin_file_sets.empty? %>
```

Comment:
Can you change `!Wayfinder.for(resource).coin_file_sets.empty?` to `Wayfinder.for(resource).coin_file_sets.present?`. The double negative spins me for a bit of a loop.

Reply:
Oh, also you should be able to just rip out `resource.try(:member_ids).present?` and just put that in the wayfinder.